src/standalone/Vegetation/radiation.jl

```diff
@@ -46,22 +47,93 @@ ClimaLSM.name(model::AbstractRadiationModel) = :radiative_transfer
 ClimaLSM.auxiliary_vars(model::BeerLambertModel) = (:apar, :par)
 ClimaLSM.auxiliary_types(model::BeerLambertModel{FT}) where {FT} = (FT, FT)
 
+"""
+    TwoStreamParameters{FT <: AbstractFloat}
+
+The required parameters for the two-stream radiative transfer model.
+$(DocStringExtensions.FIELDS)
+"""
+struct TwoStreamParameters{FT <: AbstractFloat}
+    "Leaf angle distribution function (unitless)"
+    ld::FT
+    "PAR canopy reflectance (unitless)"
+    ρ_leaf::FT
+    "leaf element transmittance"
+    τ::FT
+    "Soil albedo"
+    a_soil::FT
```

Comment:
we should think about this more. the soil albedo is specified in the soil model when they are run together, and we dont want to specify it twice. in standalone, this would need to be specified. perhaps we can discuss more on a slack call?

Reply:
We discussed offline and will follow this up in a separate PR. For now, in the soil_canopy_model constructor, perhaps we can add an @assert that checks that soil.parameters.albedo == canopy.radiative_transfer.soil_albedo?

Reply:
Went ahead and added this assertion and will follow up in a separate PR.